contracts/src/PortalRegistry.sol

```diff
@@ -190,20 +189,20 @@ contract PortalRegistry is OwnableUpgradeable {
   }
 
   /**
-   * @notice Deploys and registers a clone of default portal
+   * @notice Deploys and registers a clone of default portal V2
    * @param modules the modules addresses
    * @param name the portal name
    * @param description the portal description
    * @param ownerName name of this portal's owner
    */
-  function deployDefaultPortal(
+  function deployDefaultPortalV2(
```

Comment:
Let's still name it `deployDefaultPortal`, no?

Reply:
Done